pkg/apis/scorecard/v1alpha1/formatter.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"fmt"
+	"strings"
+)
+
```

Comment:
Add a line here to verify `ScorecardOutput` implements the interface:

```go
var _ scorecard.ScorecardFormatter = &ScorecardOutput{}
```

Ditto for v1alpha2

Reply:
I added this into the formatting.go to avoid an import cycle.